docs/sparkpost.md

```diff
@@ -0,0 +1,58 @@
+# Sparkpost
+
+**Document state: provisional, may contain errors or incomplete information. Should be reviewed by Tilmann too.**
+
+We use sparkpost for sending emails and accepting inbound emails.
+
+There are quite a few things to consider and this page hopefully will guide you to understand:
+
+* what to create in the Sparkpost admin interface
+* which DNS records you need to set
+* which variables need to be configured in the ansible config
+
+## Sparkpost admin interface
+
+Firstly, you need an account. If you don't already have one ask Nick or Tilmann for an invite. If you don't know who those people are then I think setting up Sparkpost is not your first step ;)
+
+Each deployment of Karrot needs to have the following things:
+* a subaccount
+* an API key for that subaccount (normally created when you create the subaccount)
+* an API key for the main account with two permissions enabled:
+  * **Inbound Domains: Read/Write**
+  * **Relay Webhooks: Read/Write**
+* a sending domain
+
+Keep hold of those two API keys as you'll need to store them as ansible secrets.
+
+Later, during the ansible deployment a webhook and a relay setup (inbound email configuration) will be created for you.
+
+## DNS records
+
+When you create a sending domain in Sparkpost, it will tell you which DNS record to add. You want to add the TXT one to verify domain ownership.
+
+I'm not sure if we need to also configure the bounce domain bit.
```

Comment:
I think if we don't, the sparkpost does something for us? Would be worth checking at some point.